vyxal/transpile.py

```diff
@@ -50,40 +50,41 @@ def transpile_ast(program: list[structure.Structure], indent=0) -> str:
 def transpile_single(
     token_or_struct: Union[Token, structure.Structure], indent: int
 ) -> str:
+    print("transpiling: ", token_or_struct)
     if isinstance(token_or_struct, Token):
         return transpile_token(token_or_struct, indent)
     elif isinstance(token_or_struct, structure.Structure):
         return transpile_structure(token_or_struct, indent)
     raise ValueError(
-        f"Input must be a Token or Structure,"
-        " was {type(token_or_struct).__name__}: {token_or_struct}"
+        "Input must be a Token or Structure,"
+        f" was {type(token_or_struct).__name__}: {token_or_struct}"
     )
 
 
 def transpile_token(token: Token, indent: int) -> str:
-    from helpers import indent_str, uncompress
+    from vyxal.helpers import indent_str, uncompress
 
     if token.name == TokenType.STRING:
         # Make sure we avoid any ACE exploits
```

Comment:
Just use Python's builtin `repr` here (or the `!r` format string flag); it's guaranteed to be safer.

```python
return indent_str(f"stack.append({string!r})")
```

Reply:
Fixed